### PR TITLE
add a test job to help aide in debugging

### PR DESCRIPTION
**Pull request by kannon92**
Adding a test to help aide in debugging imagefs e2e tests.

PR is https://github.com/kubernetes/kubernetes/pull/121832

### CODE REVIEW

**Comment by kannon92** [comment]
/cc @dims @SergeyKanzhelev 

I am hoping to add this so I can easily debug these tests. I can't reproduce the failures with local tests and the remote e2e tests are too slow to really make a lot of progress.

I was hoping to use this dummy job and change my PR so I can target smaller tests so I can figure out the issue.

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -2271,6 +2271,57 @@ presubmits:
...
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --deployment=node
+        - --env=KUBE_SSH_USER=core
+        - --gcp-zone=us-west1-b
+        - '--node-test-args=--container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
+        - --node-tests=true
+        - --provider=gce
+        - --test_args=--nodes=1 --focus="SeparateDiskTest" --timeout=300m
```
Comment (SergeyKanzhelev) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:2307]: is the only change here this `focus`? Anything else?

Some people need it periodically, so maybe we can have something more permanent in CI jobs that will use some generic name and people will use that name while testing.

**Review by SergeyKanzhelev — APPROVED** [review]
discussed at SIG Node CI meeting. This job hard to generalize and it is not repro locally. This is needed for troubleshooting. Approving

/lgtm
/approve

**Pull request merged**